CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+
+### Project Directory Structure
+
+- [_`src`_](https://github.com/mozilla/extension-activity-monitor/tree/master/src) - Inside _src_ the extension's code can be found (manifest.json, background.js etc.)
+  - [_`activitylog`_](https://github.com/mozilla/extension-activity-monitor/tree/master/src/activitylog) - Contains activity log page (extension page).
+  - [_`devtools`_](https://github.com/mozilla/extension-activity-monitor/tree/master/src/devtools) - Contains the invisible devtools page that creates a devtools panel which loads the activitylog.html page and filter logs with the current tab id.
+  - [_`popup`_](https://github.com/mozilla/extension-activity-monitor/tree/master/src/popup) - Contains popup (browserAction popup).
+  - [_`icons`_](https://github.com/mozilla/extension-activity-monitor/tree/master/src/icons) - Contains icons used in the extension.
+  - [_`lib`_](https://github.com/mozilla/extension-activity-monitor/tree/master/src/lib) - Contains most of the project's code. Here is a brief description of each of the files:
+    - `ext-monitor.js` - It runs in the background.
+    - `ext-listen.js` - It contains some instructions to communicate with the background.
+    - `ext-popup.js` - It runs in the popup.
+    - `ext-activitylog.js` - It runs in the Activity Log page.
+    - `save-load.js` - It contains the functionalities of saving and loading log file.
+  - [_`lib/web-component`_](https://github.com/mozilla/extension-activity-monitor/tree/master/src/lib/web-component) - It contains all the web components being used in the project.
+- [_`tests`_](https://github.com/atiqueahmedziad/extension-activity-monitor/tree/master/tests) - Contains all the test files.
+
+## Pick an issue
+
+If you are a new contributor looking for a easy bug/issue to get started, take a look at the list of [good first bugs](https://github.com/mozilla/extension-activity-monitor/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+bug"). Feel free to ask permission from maintainers by commenting on any issue that you would like to work on.
+
+## Installation
+
+To install Extension Activity Monitor, follow the given instructions [here](https://github.com/mozilla/extension-activity-monitor/blob/master/README.md#installation).
+
+## Run Test
+
+To run all the test suits once, run the following command:
```

Comment:
Remove "test suits" with just "tests".

A [test suite](https://en.wikipedia.org/wiki/Test_suite) (plural "test suites") is a collection of test cases. All tests combined is the test suite, but the individual files are just tests.